src/test/kotlin/pl/touk/krush/BookTest.kt

```diff
@@ -34,6 +34,9 @@ class BookTest {
             val persistedBook = BookTable.insert(book)
             assertThat(persistedBook.id).isNotNull()
 
+            val unused = 1
+            val TEST = 1
```

Comment:
\[detekt\] INFO: \[style/NamingConventionViolation\] All names in the codebase should be matching the naming convention of the codebase\.